packages/aws-cdk-lib/pipelines/test/compliance/assets.test.ts

```diff
@@ -396,17 +396,6 @@ describe('basic pipeline', () => {
                   Service: 'codebuild.amazonaws.com',
                 },
               },
-              {
-                Action: 'sts:AssumeRole',
-                Effect: 'Allow',
-                Principal: {
-                  AWS: {
-                    'Fn::Join': ['', [
-                      'arn:', { Ref: 'AWS::Partition' }, `:iam::${PIPELINE_ENV.account}:root`,
-                    ]],
-                  },
-                },
-              },
```

Comment:
Why does this test change when no feature flag is set?

Reply:
Since the behavior has been tested with feature flag in the other test, we can use this to test default value.